src/tests/Interop/Swift/SwiftSelfContext/SwiftSelfContext.cs

```diff
@@ -81,4 +81,26 @@ public unsafe static void TestSelfIsFrozenNonEnregisteredStruct()
         long sum = SumFrozenNonEnregisteredStruct(new SwiftSelf<FrozenNonEnregisteredStruct>(new FrozenNonEnregisteredStruct { A = 10, B = 20, C = 30, D = 40, E = 50 }));
         Assert.Equal(150, sum);
     }
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s16SwiftSelfContext24FrozenEnregisteredStructV16sumWithExtraArgs1c1dS2f_SftF")]
+    public static extern float SumFrozenEnregisteredStructWithExtraArgs(float c, float d, SwiftSelf<FrozenEnregisteredStruct> self);
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s16SwiftSelfContext27FrozenNonEnregisteredStructV16sumWithExtraArgs1f1gS2f_SftF")]
+    public static extern float SumFrozenNonEnregisteredStructWithExtraArgs(float f, float g, SwiftSelf<FrozenNonEnregisteredStruct> self);
+
+    [Fact]
+    public unsafe static void TestSelfIsFrozenEnregisteredStructWithExtraArgs()
+    {
+        float sum = SumFrozenEnregisteredStructWithExtraArgs(3f, 4f, new SwiftSelf<FrozenEnregisteredStruct>(new FrozenEnregisteredStruct { A = 10, B = 20 }));
+        Assert.Equal(37f, sum);
+    }
+
+    [Fact]
+    public unsafe static void TestSelfIsFrozenNonEnregisteredStructWithExtraArgs()
```

Comment:
It seems that this is the test which fails on the CI. So far I have determined that the error also occurs when we use non-generic SwifSelf instead. The error seems to be only occurring on the runtime x64 macos runtime_tests pipeline.

Reply:
I opened #108855 about this and disabled this test against that issue